packages/workflow/src/NodeHelpers.ts

```diff
@@ -872,7 +977,12 @@ export function getNodeWebhooksBasic(workflow: Workflow, node: INode): IWebhookD
  * @param {string} path
  * @returns {string}
  */
-export function getNodeWebhookPath(workflowId: string, node: INode, path: string, isFullPath?: boolean): string {
+export function getNodeWebhookPath(
+	workflowId: string,
+	node: INode,
+	path: string,
+	isFullPath?: boolean,
+): string {
```

Comment:
Shouldn't this be caught by the no-inferrable-types rule?

Reply:
The return type `string` is inferrable from the function body, so I thought it'd be covered by `no-inferrable-types`.

Reply:
hmmm 🤔 good question. We also have `@typescript-eslint/explicit-module-boundary-types` in place which requires that the return type of a function gets explicitly stated, which might have a higher priority.  I also think that the `no-inferrable-types` rule just applies to arguments and variables, not to function return types.